### ISSUE TITLE
add loss function weight (very important since it was used in results)

### ISSUE THREAD

**Issue by CunliangGeng** [DeepRank/deeprank#104]
Add the parameter of loss function weight to `NeuralNet.py`.

* [x] check the proper place to add it, maybe the `train` function
* [ ] check how to get the weight automatically from the ratios of classes in training data 

Reference:
1. https://pytorch.org/docs/stable/nn.html#crossentropyloss
2. https://towardsdatascience.com/speed-up-your-algorithms-part-1-pytorch-56d8a4ae7051

**Comment by sonjageorgievska** [comment]
I have it locally, we should merge

**Comment by CunliangGeng** [comment]
Already added to the code.

Close the issue now.
